app/controllers/tmdb_controller.rb

```diff
@@ -110,9 +110,13 @@ def tv_series
 
   def tv_season
     show_id = params[:show_id]
-    @season_number = params[:season_number]
+    season_number = params[:season_number]
     @series = tmdb_handler_tv_series(show_id)
-    @episodes = tmdb_handler_tv_season(show_id, @season_number)
+    @season = tmdb_handler_tv_season(
```

Comment:
This change puts the `episodes` inside the `season` object, which is the main object of the page.